internal/riverupdater/manager.go

```diff
@@ -0,0 +1,132 @@
+package riverupdater
```

Comment:
not sure about the packagename here, updater seems unnecessary. lets discuss this one later on.